bnd-shared-maven-lib/bnd.bnd

```diff
@@ -0,0 +1,59 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+group=biz.aQute.bnd
+
+Bundle-Description: Shared logic used by bnd maven plugins.
+Bundle-SymbolicName: bnd-shared-maven-lib
+Export-Package: \
+	aQute.bnd.maven.lib.configuration,\
+	aQute.bnd.maven.lib.resolve
+
+Import-Package: \
+	org.apache.maven;provider=m2e,\
```

Comment:
This is a bad way to write this `Import-Package` instruction as it is fragile with respect to future changes. It should definitely be something more like

    Import-Package: \
        org.apache.maven.*;provider=m2e,\
        org.eclipse.aether.*;provider=m2e,\
        org.codehaus.plexus.*;provider=m2e,\
        *

Reply:
got it!